docs/contributing-to-airbyte/developing-locally.md

```diff
@@ -38,6 +38,11 @@ This will build all the code and run all the unit tests.
 
 `./gradle build` creates all the necessary artifacts \(Webapp, Jars and Docker images\) so that you can run Airbyte locally.
 
+The `./gradlew build` builds the core Airbyte system, and all the connectors. This can take some time. To compile and build just the core systems:
+```bash
+export CORE_ONLY=1 && ./gradlew build
```

Comment:
wouldn't this set `CORE_ONLY` to true for every subsequent `build` command in this session too? is that the desired behavior?

Reply:
good catch. changing to `CORE_ONLY=true && ./gradlew build`

Reply:
I think you don't need the `&&` , running `CORE_ONLY=true ./gradlew build` sets the env variable for that command only

Reply:
I inserted a print statement to double check.

Interestingly,
```
➜  airbyte git:(davinchia/add-core-only-build-command) ✗ CORE_ONLY=1 && ./gradlew build --rerun-tasks
CORE_ONLY
/Users/davinchia/code/airbyte/airbyte-integrations/connectors
```

```
  airbyte git:(davinchia/add-core-only-build-command) ✗ CORE_ONLY=1 ./gradlew build --rerun-tasks

> Task :airbyte-integrations:bases:airbyte-protocol:checkPython
Using python 3.7.9 from /Users/davinchia/code/airbyte/airbyte-integrations/bases/airbyte-protocol/.venv (.venv/bin/python)
Using pip 20.2 from /Users/davinchia/code/airbyte/airbyte-integrations/bases/airbyte-protocol/.venv/lib/python3.7/site-packages/pip (python 3.7)

> Task :airbyte-integrations:bases:base-python:checkPython
Using python 3.7.9 from /Users/davinchia/code/airbyte/airbyte-integrations/bases/base-python/.venv (.venv/bin/python)
Using pip 20.2 from /Users/davinchia/code/airbyte/airbyte-integrations/bases/base-python/.venv/lib/python3.7/site-packages/pip (python 3.7)
```